src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -13402,6 +13402,41 @@ class C
                 Diagnostic(ErrorCode.ERR_ParameterIsStaticClass, "set").WithArguments("S").WithLocation(4, 11));
         }
 
+        [WorkItem(63975, "https://github.com/dotnet/roslyn/pull/63975")]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/62485")]
```

Comment:
>https://github.com/dotnet/roslyn/issues/62485 [](http://example.com/codeflow?start=22&length=45)

I think the test-plan for "Default Lambda Parameters" is to unspecific to use as the reason to skip. Let's use the original issue for that, let's make a comment in there that some scenarios are still not addressed pending the merge for the feature, and let's keep the issue open (perhaps also moving it to the next milestone) after this PR is merged.

Reply:
Alright, I added a comment to the issue and referenced that: 193b740871cb76ff50bf882adae558a126f51759. I also changed this PR's description, so it doesn't automatically close the issue.